### PR TITLE
Module 4: School library, milestone 3

### PR DESCRIPTION
**Pull request by preciousakams**
### Commit details

- created new classes
- created a has-many and belong-to relationships between rental, person, and book
- create a has-many relationship between classroom and student

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# book has-many rentals
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (Ghiftee) [inline book.rb]: - [ ] Kindly add rentals to ```attr_accessor``` so that we cannot just read but also change it.

File: person.rb [hunk 2]
```diff
@@ -1,16 +1,18 @@
 require './nameable'
+# person has-many rentals
 class Person < Nameable
+  attr_reader :id, :rentals
```
Comment (Ghiftee) [inline person.rb]: - [ ] Kindly add rentals to ```attr_accessor``` so that we cannot just read but also change it.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽 @kamzzy

Good job working on the project so far! 
✔️ Associations are set correctly
✔️ There are no linter errors
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by prolajumokeoni — APPROVED** [review]
Hi @kamzzy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**